frontend/src/components/UserInput.js

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useState } from 'react'
```

Comment:
We're guessing this is one step in making a reusable component. We like the idea, we think it's very smart.